app/src/main/java/org/wikipedia/createaccount/CreateAccountActivity.kt

```diff
@@ -289,19 +292,23 @@ class CreateAccountActivity : BaseActivity() {
         }
 
         override fun run() {
-            disposables.add(ServiceFactory.get(wiki).getUserList(userName)
-                    .subscribeOn(Schedulers.io())
-                    .observeOn(AndroidSchedulers.mainThread())
-                    .subscribe({ response ->
-                        response.query?.getUserResponse(userName)?.let {
-                            binding.createAccountUsername.isErrorEnabled = false
-                            if (it.hasBlockError) {
-                                handleAccountCreationError(it.error)
-                            } else if (!it.canCreate) {
-                                binding.createAccountUsername.error = getString(R.string.create_account_name_unavailable, userName)
-                            }
+            lifecycleScope.launch {
```

Comment:
Would it be possible to use `CoroutineExceptionHandle`r instead of a `try catch`?

Reply:
Applied in cde801e500c1b1ce72193e19273fc749cc299b62